sdk/identity/azure-identity/azure/identity/aio/_credentials/azd_cli.py

```diff
@@ -2,7 +2,7 @@
 # Copyright (c) Microsoft Corporation.
 # Licensed under the MIT License.
 # ------------------------------------
-import asyncio
+import asyncio  # pylint: disable=do-not-import-asyncio
```

Comment:
Is this by design?

Reply:
I believe so. The `asyncio` library provides the simplest way to run subprocesses asynchronously.